lib/galaxy/webapps/galaxy/api/quotas.py

```diff
@@ -145,3 +144,11 @@ def undelete( self, trans, id, **kwd ):
             return self._undelete_quota( quota )
         except ActionInputError as e:
             raise HTTPBadRequest( detail=str( e ) )
+
+
+def get_quota( trans, id ):
```

Comment:
Awesome. Thanks for catching this. I assumed that this is called due to the previous inheritance to the shared admin controller.